README.md

```diff
@@ -68,12 +68,20 @@ The easiest way to install Spyder is:
 
 ### On Windows:
 
-- Or using one of these scientific Python distributions:
+- You can use `pip`:
+
+    pip install spyder
```

Comment:
> Is it because your work is sponsored by Continuum or there is another reason?

No, it is because it's far more easier to download and run a point-and-click installer, than:

1. Downloading Python
2. Opening a Terminal and running `pip`.
3. Not knowing how to run Spyder (because our wheels don't install menu entries).

So this method is really for power users. Jupyter guys also recommend the same:

http://jupyter.readthedocs.io/en/latest/install.html#id4

> Where are those wheels?

Where they should be, in PyPI:

https://pypi.python.org/pypi/spyder/3.0.0b3

----

After giving it more thought, I'm inclined to remove this block. We already mention how to use `pip` to install Spyder in a cross-platform way, so we don't need to repeat it here:

https://github.com/spyder-ide/spyder#cross-platform-way-from-source

So please remove it :-)